src/compiler/compiler.rs

```diff
@@ -256,8 +255,8 @@ where
         let duration = start.elapsed();
         let outputs = compilation
             .outputs()
-            .map(|(key, path)| (key.to_string(), cwd.join(path)))
-            .collect::<HashMap<_, _>>();
+            .map(|(key, path, optional)| (key.to_string(), cwd.join(path), optional))
+            .collect::<Vec<_>>();
```

Comment:
Shouldn't this become a `HashMap<key,(path, optional)>`?

Reply:
`extract_objects` accepts `T: IntoIterator<Item = (key, path, optional)>`, so it cannot be `HashMap` now.
And this `outputs` variable was not used as HashMap anywhere.